### PR TITLE
Delete resource

### PR DESCRIPTION
**Pull request by Thiruni**

### CODE REVIEW

File: packages/frontend/src/api/resources.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
-import { get, post } from 'utils/http';
+import { get, post, del } from 'utils/http';
 
 const namespace = 'core';
 
 export const create = (resource) => post(`/${namespace}/resource`, resource);
 export const fetch = (id) => get(`/${namespace}/resource/${id}`);
 export const fetchAll = () => get(`${namespace}/resource`);
+export const clear = (id) => del(`/${namespace}/resource/${id}`);
```
Comment (thinkholic) [inline packages/frontend/src/api/resources.js]: Let's change `clear` to `remove`.

File: packages/frontend/src/screens/Resources/index.js [hunk 2]
```diff
@@ -109,7 +119,7 @@ export default function Resources() {
                   <AddDocumentIcon
                     onClick={onClickResourceHandler.bind(null, _id)}
                   />
-                  <DeleteIcon onClick={() => {}} />
+                  <DeleteIcon onClick={onDeleteclick.bind(null,_id)} />
```
Comment (thinkholic) [inline packages/frontend/src/screens/Resources/index.js]: Alternatively you can use like this; `onClick={() => onDeleteclick(id)}`

**Review by thinkholic — CHANGES_REQUESTED** [review]
@Thiruni Nice job. :)

added a few comments for minor changes. Also, just ran `yarn format` in `packages/frontend` once

File: packages/frontend/src/api/resources.js (hunk 1 above)
Comment (Thiruni) [reply packages/frontend/src/api/resources.js]: This is fixed

File: packages/frontend/src/screens/Resources/index.js (hunk 2 above)
Comment (Thiruni) [reply packages/frontend/src/screens/Resources/index.js]: This is fixed

**Comment by Thiruni** [comment]
@heimdallrhg Requested changes are done could you please review it again

**Review by thinkholic — APPROVED** [review]

**Pull request merged**